sample/mpp-library/src/iosMain/kotlin/com/icerockdev/library/CoroutineScope.kt

```diff
@@ -0,0 +1,9 @@
+package com.icerockdev.library
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+
+fun createCoroutineScope() {
+    CoroutineScope(Dispatchers.Main + SupervisorJob())
+}
```

Comment:
what here is do? here no return value at all - function return unit and you just create new object inside  without return to outside...